src/components/CopyTextToClipboard.tsx

```diff
@@ -1,44 +1,40 @@
-import PropTypes from 'prop-types';
 import React, {useCallback} from 'react';
+import {StyleProp, TextStyle} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
 import Clipboard from '@libs/Clipboard';
 import * as Expensicons from './Icon/Expensicons';
 import PressableWithDelayToggle from './Pressable/PressableWithDelayToggle';
-import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import {WithLocalizeProps} from './withLocalize';
 
-const propTypes = {
+type CopyTextToClipboardProps = WithLocalizeProps & {
```

Comment:
why you are adding WithLocalizeProps when you are removing withLocalize hoc from this file?

Reply:
Right, silly mistake 😅